docs/JSON-RPC-API/Authentication.md

```diff
@@ -0,0 +1,88 @@
+
+* All API methods, specify `["*:*"]`.
+* All API methods in an API group, specify `["<api_group>:*"]`. For example, `["eth:*"]`. 
+* Specific API methods, specify `["<api_group>:<method_name>"]`. For example, `["admin:peers"]`.
+
+To explicitly specify a user cannot access any methods, include the user with an empty permissions list (`[]`). 
+Users with an empty permissions list and users not included in the credentials file cannot access any JSON-RPC
+methods. 
+    
+## Enabling Authentication 
+ 
+Use the [` --rpc-http-authentication-enabled`](../Reference/Pantheon-CLI-Syntax.md#rpc-http-authentication-enabled) or 
+ [`--rpc-ws-authentication-enabled`](../Reference/Pantheon-CLI-Syntax.md#rpc-ws-authentication-enabled)
+ options to require authentication for the JSON-RPC API.
+  
+Use the [`--rpc-http-authentication-credentials-file`](../Reference/Pantheon-CLI-Syntax.md#rpc-http-authentication-credentials-file)
+and [`--rpc-ws-authentication-credentials-file`](../Reference/Pantheon-CLI-Syntax.md#rpc-ws-authentication-credentials-file) 
+options to specify the [credentials file](#credentials-file).  
+
+## Obtaining an Authentication Token 
+
+To obtain an authentication token, make a request to the `/login` endpoint with your username and password. 
+
+!!! example
+    ```bash tab="curl HTTPS request"
+    $ curl -X POST --data '{"username":"username1","password":"pegasys"}' <JSON-RPC-https-endpoint:port>/login
+    ```
+    
+    ```json tab="JSON result"
+    {"token":"eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJwZXJtaXNzaW9ucyI6WyIqOioiXSwidXNlcm5hbWUiOiJ1c2VyMiIsImlhdCI6MTU1MDQ2MDYwNCwiZXhwIjoxNTUwNDYwOTA0fQ.l2Ycqzl_AyvReXBeUSayOlOMS_E8-DCuz3q0Db0DKD7mqyl6q-giWoEtfdWzUEvZbRRi2_ecKO3N6JkXq7zMKQAJbVAEzobfbaaXWcQEpHOjtnK4_Yz-UPyKiXtu7HGdcdl5Tfx3dKoksbqkBl3U3vFWxzmFnuu3dAISfVJYUNA"}
+    ``` 
+
+Authentication tokens expire 5 minutes after being generated. It is necessary to generate a new authentication 
+token if access is required after token expiration.     
+    
+## Using an Authentication Token to Make Requests 
+
+Specify the authentication token as a `bearer` token in the JSON-RPC request header. 
+
+If using Postman, specify a key:value pair in the _Headers_ tab where the key is `bearer` and the value is the token 
+generated by the [`login` request](#obtaining-an-authentication-token): 
+
+![Postman Request](../images/PostmanAuthToken.png)
+
+If using curl, specify the `bearer` in the header. 
+
+!!! example "curl Request with Authentication"
+    ```bash
+    curl -X POST -H 'bearer: eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJwZXJtaXNzaW9ucyI6WyIqOioiXSwidXNlcm5hbWUiOiJ1c2VyMiIsImlhdCI6MTU1MDQ2MTQxNiwiZXhwIjoxNTUwNDYxNzE2fQ.WQ1mqpqzRLHaoL8gOSEZPvnRs_qf6j__7A3Sg8vf9RKvWdNTww_vRJF1gjcVy-FFh96AchVnQyXVx0aNUz9O0txt8VN3jqABVWbGMfSk2T_CFdSw5aDjuriCsves9BQpP70Vhj-tseaudg-XU5hCokX0tChbAqd9fB2138zYm5M' -d '{"jsonrpc":"2.0","method":"net_listening","params":[],"id":1}' https://localhost:8545
```

Comment:
I think before the example with real data, we could add one with placeholders. It makes it easier to see the structure of the request:

```
curl -X POST -H 'Bearer: <JWT_TOKEN>' -d '{"jsonrpc":"2.0","method":"<API_METHOD>","params":[],"id":1}' https://localhost:8545
```

Reply:
Added